style.css

```diff
@@ -283,3 +298,201 @@ header {
   line-height: 20px;
   color: #253858;
 }
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  height: 48.6875rem;
+  background-color: #6070ff;
+  gap: 8px;
```

Comment:
Kindly consider adding a background image shape to make it resemble the design in Figma. 😁 Since the purpose of this project is to code a design that matches the provided design. 🚀

![Screenshot 2023-11-30 213223](https://github.com/Frederick-Ye/portfolio-project/assets/102011195/3c8243f7-1ce7-4fc4-95c9-38ce5a7aef8f)